### DISCUSSION TITLE
Include BoDi to Reqnroll repo

### DISCUSSION THREAD

**Discussion by mbhoek** [reqnroll/Reqnroll#40] [Contributor Q&A] [answered]
[BoDi](https://github.com/specflowoss/bodi), the simple IoC container used in Reqnroll, is still in the SpecFlowOSS organization afaict.

Should this be moved to the Reqnroll organization as well?

**Comment by gasparnagy** [comment, marked as answer]
It's soo good to see that you are faster processing my TODO list than myself. 

The short answer is: yes. 

My suggestion would be to include it to the main Reqnroll repo tough. (We can still make a separate NuGet package of that as Reqnroll.BoDi.) Having it in the same repo makes development and debugging (especially paraellel isssues) much easier. I would put it in a root folder "Reqnroll.BoDi" (like "Reqnroll.Utils") and include it's unit tests to the Tests/Reqnroll.RuntimeTests (maybe in a BoDi subfolder). I would not worry about the history in this case - it is a simple code - just make a code drop as it is right now.

**Reply by mbhoek** [reply]
Ok, I'm having a go at this (I need BoDi to port my plugin anyway). Adding the BoDi project was pretty easy, adding the tests (which are still NUnit based and use something different to FluentAssertions) is a little bit more complicated. Doing the work [here](https://github.com/reqnroll/Reqnroll/pull/80).

**Reply by gasparnagy** [reply]
@mbhoek perfect, thx. I have added you as to the main repo, so next time you can do it in a simple branch as well.

**Reply by mbhoek** [reply]
@gasparnagy Just to be clear; you are allowing me to branch straight off `Reqnroll` instead of having to fork it first?

**Reply by gasparnagy** [reply]
@mbhoek yes, exactly. I guess it is easier that way and for more complex PR it is easier to collaborate with others that way.

**Comment by gasparnagy** [comment]
The related PR #80 is done & merged. Thx @mbhoek. 

Closing this discussion.
